flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -2546,7 +2557,9 @@ void takeSnapshot() throws Exception {
 
 			// create hard links of living files in the snapshot path
 			try (Checkpoint checkpoint = Checkpoint.create(stateBackend.db)) {
-				checkpoint.createCheckpoint(localBackupDirectory.getDirectory().getPath());
+				LOG.info("Checkpoint path is {}.", localBackupDirectory.getDirectory().getPath());
+				String path = Utils.getPathAsString(localBackupDirectory.getDirectory());
```

Comment:
Why not prepare this string just once in the evry beginning and use it for the log message as well?